### DISCUSSION TITLE
Aggregated images using -d but can't --continue

### DISCUSSION THREAD

**Discussion by TheGigaQuant** [sepinf-inc/IPED#2370] [General]
Hello, I used the command -d to aggregate multiples .dd images and I ran out of memory.
Now, when I launch the SearchApp, I have the following message  
![image](https://github.com/user-attachments/assets/96cf12bc-7bc5-4ee6-8f9c-cda615d4ad4b)

But when I relaunch the same command with --continue, I have the following ;

`.\iped.exe -d "I:\Backup\EXCEL STORE\EXCEL STORE.dd" -d "I:\Backup\HD252KJ\S0NJJDWQ112147.dd" -d "I:\Backup\MK3276GSX\MK3276GSX.dd" -d "I:\Backup\STA207MC20WD6B\STA207MC20WD6B.dd" -d "I:\Backup\WD AV-25\WX51A544C1DV.dd" -d "I:\Backup\WD1600AAJS\WD1600AAJS - HDD Backup.dd" --continue
Error: You cannot use --append, --continue or --restart with an inexistent or invalid case folder.`

An idea ?
Thanks 👍

**Comment by lfcnassif** [comment]
It maybe is not be the same command, the -o parameter is missing.

**Comment by wladimirleite** [comment]
1. The "Processing didn't finish successfully" is expected, as the processing didn't finish.
2. You can omit the `-o` (case output folder) parameter, but I usually don't like to do that.
3. When `-o` is omitted, the case is placed in the parent folder of the first evidence ("I:\Backup\EXCEL STORE"). It would make more sense to explicitly use something like `-o "I:\Backup"`.
4. The `--continue` needs the explicit `-o`. Try to add `-o "I:\Backup\EXCEL STORE"` (or wherever the case is) to your command.
